### ISSUE TITLE
crate v1.4.2 does not include the bin/ folder in project root

### ISSUE THREAD

**Issue by Congee** [doy/rbw#81]
Thank you @doy for a prompt fix 😊  Some downstream packages such as [nixpkgs](https://github.com/NixOS/nixpkgs/blob/86045424d2bafbb130d4ea3d9522cebe695766cd/pkgs/tools/security/rbw/default.nix#L47-L51) break due to lack of bin/git-credential-rbw.

Below is a list of files downloaded from crates.io
```
rbw-1.4.2/.cargo_vcs_info.json
rbw-1.4.2/CHANGELOG.md
rbw-1.4.2/Cargo.lock
rbw-1.4.2/Cargo.toml
rbw-1.4.2/Cargo.toml.orig
rbw-1.4.2/LICENSE
rbw-1.4.2/README.md
rbw-1.4.2/src/actions.rs
rbw-1.4.2/src/api.rs
rbw-1.4.2/src/bin/rbw/actions.rs
rbw-1.4.2/src/bin/rbw/commands.rs
rbw-1.4.2/src/bin/rbw/main.rs
rbw-1.4.2/src/bin/rbw/sock.rs
rbw-1.4.2/src/bin/rbw-agent/actions.rs
rbw-1.4.2/src/bin/rbw-agent/agent.rs
rbw-1.4.2/src/bin/rbw-agent/daemon.rs
rbw-1.4.2/src/bin/rbw-agent/debugger.rs
rbw-1.4.2/src/bin/rbw-agent/main.rs
rbw-1.4.2/src/bin/rbw-agent/sock.rs
rbw-1.4.2/src/cipherstring.rs
rbw-1.4.2/src/config.rs
rbw-1.4.2/src/db.rs
rbw-1.4.2/src/dirs.rs
rbw-1.4.2/src/edit.rs
rbw-1.4.2/src/error.rs
rbw-1.4.2/src/identity.rs
rbw-1.4.2/src/json.rs
rbw-1.4.2/src/lib.rs
rbw-1.4.2/src/locked.rs
rbw-1.4.2/src/pinentry.rs
rbw-1.4.2/src/prelude.rs
rbw-1.4.2/src/protocol.rs
rbw-1.4.2/src/pwgen.rs
rbw-1.4.2/src/wordlist.rs
```

**Comment by doy** [comment]
oh, i didn't realize those were actually getting packaged from the crate itself - i'll replace them

**Comment by doy** [comment]
fixed in 1.4.3
